components/tyk-bootstrap/values.yaml

```diff
@@ -145,3 +145,6 @@ bootstrap:
     postInstallPodAnnotations: {}
     # preDeletePodAnnotations specifies annotations to be added to Tyk Bootstrap pre-delete hook's job pods.
     preDeletePodAnnotations: {}
+
+  # podLabels specifies labels to be added in bootstrap Pod
+  podLabels: {}
```

Comment:
for annotations, we have defined separate fields like postInstallPodAnnotations, preDeletePodAnnotations and preInstallPodAnnotations. Maybe we can adopt similar approach for labels as well. what do you think

Reply:
Sure we can